app/components/provider_interface/new_reference_with_feedback_component.rb

```diff
@@ -69,6 +73,13 @@ def feedback_row
       }
     end
 
+    def confidentiality_row
+      {
+        key: 'Can this reference be shared with the candidate?',
```

Comment:
Should we use translations in this file?

Reply:
I didn't use translations in this case because I followed the existing pattern in this file. If we were to put this in translations, it would make sense to move all text in this file to translations for consistency. I personally find it a bit overkill to use translations for text that's only used in one place, especially if there aren't any immediate plans to reuse it elsewhere or support multiple languages. 

I know we do use it everywhere in Apply though. What are your thoughts?